test/verifiedTests/analysis/testConnectedComponents/testConnectedComponents.m

```diff
@@ -27,31 +27,29 @@
 testdir = fileparts(which('testConnectedComponents.m'));
 cd(testdir)
 
-IPT = 'Image Processing Toolbox';
-IPT_Lic = 'Image_Toolbox';
-v = ver;
+requiredToolboxes = {'image_toolbox'};
```

Comment:
Would it make sense to declare a default cell array with all the toolboxes and all the solvers?

Reply:
In general: yes. But I don't see how this relates to this particular test.
For each test, you will need the specific toolboxes.

Reply:
I agree that a default cell array is not applicable in the case of the present test. However, I think it would be appropriate to define a default cell array with all the required toolboxes.